source/Core/Drivers/Font.h

```diff
@@ -145,6 +145,14 @@ const uint8_t disconnectedTipIcon[] = {
     //
 };
 
+// 16 x 16
+const uint8_t brightnessIcon[]
+    = {0x80, 0x86, 0x8E, 0x9C, 0x18, 0xC0, 0xE0, 0xEF, 0xEF, 0xE0, 0xC0, 0x18, 0x9C, 0x8E, 0x86, 0x80, 0x01, 0x61, 0x71, 0x39, 0x18, 0x03, 0x07, 0xF7, 0xF7, 0x07, 0x03, 0x18, 0x39, 0x71, 0x61, 0x01};

```

Comment:
@alvinhochun
Good evening Sir,

1. may I please ask you, what piece of software you are using to convert/create pictures into the *hex format*?

2. I have stumbled across this fine piece of software [Piskel](https://github.com/piskelapp/piskel), which is already capable of exporting *C files*, but it unfortunately does not offer *`hex`* as of now.
Since this project (Piskel) is *open source* and you have the programming skills, I dare to ask you, if it would be possible for you to fork this and to implement the *hex export* functionality?
## Please don't get me wrong, I am not pushing you, just asking, if you are willing.

thanks in advance
kind regards

Reply:
Here's the directory to the tool: https://github.com/Ralim/IronOS/blob/master/Translations/BitmapEditor.html

Reply:
Thank you Sir.